### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by rodja** [elimintz/justpy#260] [Ideas]
In https://github.com/elimintz/justpy/pull/258#issuecomment-848446754 Eli asked me about docker and simplified deployment:

> Since you have experience with docker, perhaps you could define a container that would make it simple for people to deploy JustPy? Any ideas in general on how to make deployment as simple as possible would be much appreciated. A command that takes a JustPy project and deploys it to one of the cloud service providers would be a huge improvement to the project.

@elimintz would you like the docker image as part of this project or rather as a separate repository?

**Comment by elimintz** [comment]
I would prefer it to be part of the project. This is now a feature people expect from web frameworks.
